### ISSUE TITLE
No Documentation for displayName With SWC

### ISSUE THREAD

**Issue by machineghost** [styled-components/styled-components-website#907]
## Environment

`npx envinfo --system --binaries --npmPackages styled-components,babel-plugin-styled-components --markdown`

This is irrelevant to the bug, but:

```
## System:
 - OS: Linux 5.15 Linux Mint 21.1 (Vera)
 - CPU: (8) x64 Intel(R) Core(TM) i7-4810MQ CPU @ 2.80GHz
 - Memory: 4.55 GB / 23.39 GB
 - Container: Yes
 - Shell: 5.1.16 - /bin/bash
## Binaries:
 - Node: 19.7.0 - /usr/bin/node
 - npm: 9.5.0 - /usr/bin/npm
## npmPackages:
 - styled-components: ^5.3.9 => 5.3.9 
```

## Reproduction

There's no documentation on using SWC with Styled Components Tooling.

## Steps to reproduce

* Go to https://styled-components.com/docs/tooling.
*  Try to find instructions on configuring tooling (eg. `displayName`) with SWC (instead of Babel).
* Find none.

## Expected Behavior

Documentation exists.

## Actual Behavior

Documentation doesn't exist.

To be clear, it's not like the whole page needs to be duplicated for SWC ... but just providing *some* explanation of how to enter configuration options with SWC would be helpful.

**Comment by probablyup** [comment]
Looks like there's a nice plugin for that: https://www.npmjs.com/package/@swc/plugin-styled-components

If someone would like to put together a quick page, I'd appreciate it!

**Comment by Cyanhead** [comment]
Hi @quantizor, looks like this issue has been fixed in #967. This issue should be closed!

**Comment by quantizor** [comment]
Thanks :)
